internal/idp/keycloak/types.go

```
@@ -243,3 +243,47 @@ func fromKeycloakAuthorizationResource(kcResource *keycloakAuthorizationResource
+// Identity providers are configured at the realm level and can be linked to specific organizations.
+type keycloakIdentityProvider struct {
+	Alias       string            `json:"alias,omitempty"`
+	DisplayName string            `json:"displayName,omitempty"`
+	InternalID  string            `json:"internalId,omitempty"`
+	ProviderID  string            `json:"providerId,omitempty"` // "ldap", "oidc", "saml", etc.
+	Enabled     bool              `json:"enabled,omitempty"`
+	Config      map[string]string `json:"config,omitempty"` // Provider-specific configuration
+}
+
+// Conversion functions for identity providers
+func toKeycloakIdentityProvider(idpProvider *idp.IdentityProvider) *keycloakIdentityProvider {
```

golangci-lint (2.12.2)

[error] func toKeycloakIdentityProvider is unused

(unused)